doc/USERMANAGEMENT.md

```diff
@@ -51,12 +51,12 @@ JupyterHub configuration allows you to give access to users/groups through templ
 * `uid={username},cn=admin,dc=ldap,dc=cetic,dc=be`
 * `uid={username},ou=developers,dc=ldap,dc=cetic,dc=be`
 
-where `{username}` will be overwrought by the value the user passes as username in the authentication screen. Let's suppose we only have those two templates, when the user david passes his name for authentication, for him to successfully sign on, his entry should be one of the following:
+where `{username}` will be overwrought by the value the user passes as username in the authentication screen. Let's suppose we only have those two templates. When the user david passes his name for authentication, for him to successfully sign on, his entry should be one of the following:
 
 * `uid=david,ou=admins,dc=ldap,dc=cetic,dc=be`
 * `uid=david,ou=developers,dc=ldap,dc=cetic,dc=be`
 
-which means if david isn't in the developers group or the admins group, he will not be able to sign in.
+which means if david is not in the developers group or the admins group, he will not be able to sign in.
 
 A sample configuration can be found in the `jupyterhub:auth` section of the default FADI [`values.yaml` file](https://github.com/cetic/helm-fadi/blob/master/values.yaml)
```

Comment:
use relative path for link

Reply:
the values.yaml file isn't in this repo that's why i used the fullpath.